public/styles.css

```diff
@@ -1073,6 +1073,7 @@ input[type=number]::-webkit-outer-spin-button {
 .mobile-content .trade-params > div {
     margin-right: .5rem;
     min-width: 10rem;
```

Comment:
My reasoning is simple and possibly flawed, having min-width will make sure all control have similar sizes, which would looks better.

Is this reasoning sound? or is the current approach problematic?